roles/postgresql/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+postgresql_image: docker.io/library/postgres:16
```

Comment:
We should stick with 13 for now to match our nightly deployment.

Reply:
This image is debian based, which means we would mix-n-match base OS which I think we said we do not want to do. Give this a try -- https://quay.io/repository/sclorg/postgresql-13-c9s